x-pack/plugins/event_log/server/es/documents.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EsNames } from './names';
+import mappings from '../../generated/mappings.json';
+
+export function getIndexTemplate(esNames: EsNames, ilmExists: boolean) {
```

Comment:
Would there be a need for these variables to be configurable?

Reply:
`esNames` is needed - someone else calculates the names of all the things, based on the kibana index prefix.  `ilmExists` won't be needed longterm, is today since you usually can't create an ILM policy, but in the future you always should be able to.

Reply:
per conversation with mike, he was wondering specifically about whether

```
      number_of_shards: 1,
      number_of_replicas: 1,
```

Should somehow be configurable.  I think we actually need to set replicas to 0 to start with.  But beyond that, this seems like it would only be needed for "large" deployments that ended up with a lot of logging.

Probably worth creating a section in the README about "customizations that can be made by customers", including this, and updating the ILM policy.